match_merger.py

```diff
@@ -11,22 +12,42 @@ class MatchMerger(Executor):
     The MatchMerger merges the results of shards by appending all matches..
     """
 
-    def __init__(self, default_traversal_paths: Tuple[str, ...] = ('r',), **kwargs):
+    def __init__(
+        self,
+        default_traversal_paths: Tuple[str, ...] = ('r',),
```

Comment:
```suggestion
        traversal_paths: Tuple[str, ...] = ('r',),
```

And change it anywhere else it is used